optimize/backend/src/main/java/io/camunda/optimize/service/CamundaLicenseService.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.optimize.service;
+
+import io.camunda.service.license.CamundaLicense;
+import org.springframework.stereotype.Component;
+
+@Component
+public class CamundaLicenseService {
+
+  private final CamundaLicense license;
+
+  public CamundaLicenseService() {
+    final String camundaLicense = System.getenv("CAMUNDA_LICENSE_KEY");
```

Comment:
Let's have a constant for this, if one doesn't already exist

Reply:
One was added with https://github.com/camunda/camunda/pull/21236

I've taken out the `String` and am using the constant now